.swift-format

```diff
@@ -10,9 +10,9 @@
     "rules": {
         "AlwaysUseLowerCamelCase": false,
         "AmbiguousTrailingClosureOverload": false,
-        "NoBlockComments": false,
+        "NoBlockComments": true,
         "OrderedImports": true,
-        "UseLetInEveryBoundCaseVariable": false,
-        "UseSynthesizedInitializer": false
+        "UseLetInEveryBoundCaseVariable": true,
+        "UseSynthesizedInitializer": true
```

Comment:
I’ve noticed that @ahoppen mentioned not having a strong opinion here, so I’d like to share my perspective. In my projects, when a case in a closure has multiple parameters, and I want to designate one parameter as a variable (var) and the others as constants (let), applying this rule makes sense since you can see what is mutable and what's not–don't know if this statment makes sense to u XD hope so. 

Code example:
```swift
enum Foo {
  case bar(name: String: age: Int)
}

let foo = Foo.bar(name: "Tom", age: 22)

if case .bar(let name, var age) = foo {
  age += 1
  // do sth with increased age and not mutated name? 
}
```

Reply:
This is one of the reasons I prefer `UseLetInEveryBoundCaseVariable` as well. Wrapping the whole pattern with `let/var` forces you into an all-or-nothing situation with the bindings. Once we have other kinds of bindings like `borrowing`, `inout`, etc., I think it'll be even more common for those to differ within a payload.

The other reason for this rule is that having the binding specifier immediately adjacent to the binding makes it easier to see what the intended outcome is, and avoids mistakes like this:

```swift
let age = 20
if case let .bar(name, age) = foo { ... }
// oops I wanted to bind `name` and match `age`, not create a
// new `age` binding

if case .bar(let name, age) = foo { ... }
// much clearer about which one is a new binding and which
// is an expression to be matched
```